Failed to compile on aarch64-darwin

```
[1/2] Compiling C++ object src/nix-eval-jobs.p/nix-eval-jobs.cc.o
[2/2] Linking target src/nix-eval-jobs
FAILED: src/nix-eval-jobs
clang++  -o src/nix-eval-jobs src/nix-eval-jobs.p/nix-eval-jobs.cc.o -Wl,-dead_strip_dylibs -Wl,-headerpad_max_install_names -Wl,-undefined,error /nix/store/7dh9a3rlriwdp8j9120vwwp5s3aqr8yd-nix-2.8.0/lib>
Undefined symbols for architecture arm64:
  "nix::MixEvalArgs::getAutoArgs(nix::EvalState&)", referenced from:
      Proc::Proc(std::__1::function<void (nix::EvalState&, nix::Bindings&, nix::AutoCloseFD&, nix::AutoCloseFD&)> const&)::'lambda'()::operator()() const in nix-eval-jobs.cc.o
  "nix::MixEvalArgs::MixEvalArgs()", referenced from:
      MyArgs::MyArgs() in nix-eval-jobs.cc.o
  "nix::lookupFileArg(nix::EvalState&, std::__1::basic_string_view<char, std::__1::char_traits<char> >)", referenced from:
      releaseExprTopLevelValue(nix::EvalState&, nix::Bindings&) in nix-eval-jobs.cc.o
ld: symbol(s) not found for architecture arm64
```

2e3405f2d10358fd1dde5712b7b61ee7243462d3, NixOS/nixpkgs@3c0f57e36ed0cf9947281e3b31f1bebb7ce5d4a1.

--- a/flake.nix
+++ b/flake.nix
@@ -17,7 +17,7 @@
           let
             mkVariant = nix: (packages.nix-eval-jobs.override {
               # TODO: fix to stable after next nix release
-              nix = pkgs.nixUnstable;
+              nix = pkgs.nix;
               #inherit nix;
             }).overrideAttrs (_: {
               name = "nix-eval-jobs-${nix.version}";
@@ -46,9 +46,7 @@
               touch $out
             '';
 
-            # TODO fix to unstable in next release
-            build = mkVariant pkgs.nixUnstable;
-            #build = mkVariant pkgs.nix;
+            build = mkVariant pkgs.nix;
             build-unstable = mkVariant pkgs.nixUnstable;
           };
 

--- a/meson.build
+++ b/meson.build
@@ -6,6 +6,7 @@ project('nix-eval-jobs', 'cpp',
 nix_main_dep = dependency('nix-main', required: true)
 nix_store_dep = dependency('nix-store', required: true)
 nix_expr_dep = dependency('nix-expr', required: true)
+nix_cmd_dep = dependency('nix-cmd', required: true)
 threads_dep = dependency('threads', required: true)
 nlohmann_json_dep = dependency('nlohmann_json', required: true)
 boost_dep = dependency('boost', required: true)

--- a/src/meson.build
+++ b/src/meson.build
@@ -7,6 +7,7 @@ executable('nix-eval-jobs', src,
              nix_main_dep,
              nix_store_dep,
              nix_expr_dep,
+             nix_cmd_dep,
              boost_dep,
              nlohmann_json_dep,
              threads_dep

